### DISCUSSION TITLE
Time Shooter 3: SWAT

### DISCUSSION THREAD

**Discussion by giscus** [astrogames06/astrogames06.github.io#6] [Q&A]
# Time Shooter 3: SWAT

https://astrogames06.github.io/game.html

**Comment by astrogames06** [comment]
good game

**Reply by 584997-ai** [reply]
does not load

**Reply by 584997-ai** [reply]
and why is there an egg

**Comment by astrogames06** [comment]
<img src="https://static01.nyt.com/images/2019/02/05/world/05egg/15xp-egg-promo-superJumbo-v2.jpg">
